models/CreateNewPatientRequest.js

```diff
@@ -0,0 +1,83 @@
+/*
+
+Patient {
+
+  family : string
+  given : string
+  suffix : string
+  phone : {
+    value : string
+    use : enum (home, work, temp, old, mobile)
+  }
+  email : array of strings
+  gender : enum (male, female, other, unknown)
+  birthdate : string
+  race : enum (American Indian or Alaskan Native, Asian, Native Hawaiian or Other Pacific Islander, Black or African-American, White, Other Race)
+  ethnicity: enum (Hispanic, Not Hispanic)
+  permissionToFollowup: boolean
+  address: {
```

Comment:
We need to add "address use" (or address type).  Options are:  home, business, temporary

Reply:
Done